corehq/form_processor/models.py

```diff
@@ -351,6 +367,9 @@ def to_json(self, include_attachments=False):
         data = dict(serializer.data)
         data['history'] = [dict(op) for op in data['history']]
         data['backend_id'] = 'sql'
+        # Remove these additional SQL only fields from JSON
+        for field in ['time_end', 'time_start', 'commcare_version', 'build_version']:
+            data.pop(field, None)
```

Comment:
I thought we'd agreed to keep these in the JSON so that the pillow would have access to them?

Reply:
I think we were of opposite opinion but each thought that we agreed, I will add it back.

Reply:
Instead of removing them here you should add these to the 'exclude' option of the serializer: https://github.com/dimagi/commcare-hq/blob/3da0cf07f056f22a7e56ea5d4574d718ff57ff5c/corehq/form_processor/serializers.py#L53